Services/Music_Portal.Services.Interfaces/Models/TopArtistsResponseLastFm.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Interfaces.Models
+﻿namespace Services.Interfaces.Models
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Interfaces.Models`